MAP/src/components/CountryInfo/CountryInfo.js

```diff
@@ -53,7 +53,7 @@ const CountryInfo = (props) => {
       <div
         style={{
           color: `${props.dark ? 'white' : 'black'}`,
-          backgroundColor: `${props.dark ? '#333333' : '#e0e0e0'}`,
+          // backgroundColor: `${props.dark ? '#333333' : '#e0e0e0'}`,
```

Comment:
maybe if these style isn't needed the line should be deleted instead of commented out??